lib/SearchAndSort/SearchAndSort.js

```diff
@@ -257,6 +258,7 @@ class SearchAndSort extends React.Component {
     selectedIndex: '',
     syncQueryWithUrl: false,
     hasNewButton: true,
+    regExpForQuery: '',
```

Comment:
Is there a PR in ui-inventory to pass this prop to the component?

Reply:
@BogdanDenis  I will create a PR in inventory as soon as that one will be merged